apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/TransformerUtilsV2.java

```diff
@@ -190,10 +191,36 @@ static CodeableConcept createCodeableConcept(String codingSystem, String codingC
   static CodeableConcept createCodeableConcept(
       String codingSystem, String codingVersion, String codingDisplay, String codingCode) {
     CodeableConcept codeableConcept = new CodeableConcept();
+    if (codingSystem == IcdCode.CODING_SYSTEM_ICD_10) {
+      AddCodingToCodeableConcept(
+          codeableConcept,
+          IcdCode.CODING_SYSTEM_ICD_10_MEDICARE,
+          codingVersion,
+          codingDisplay,
+          codingCode);
+    }
+    AddCodingToCodeableConcept(
+        codeableConcept, codingSystem, codingVersion, codingDisplay, codingCode);
+    return codeableConcept;
+  }
+
+  /**
+   * @param codeableConcept the {@link CodeableConcept} to use
+   * @param codingSystem the {@link Coding#getSystem()} to use
+   * @param codingVersion the {@link Coding#getVersion()} to use
+   * @param codingDisplay the {@link Coding#getDisplay()} to use
+   * @param codingCode the {@link Coding#getCode()} to use
+   * @return
+   */
+  static void AddCodingToCodeableConcept(
```

Comment:
first letter of this method should be lowercase

Reply:
Fixed